roles/bugs/tasks/main.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Install web configuration
+  template:
+    dest: /etc/nginx/conf.d/{{ server_name }}.conf
+    src: nginx.conf
+  notify: restart nginx
+
+- command: semanage fcontext -a -t httpd_user_rw_content_t /srv/{{ server_name }}/html/(/.*)?
+
+- name: Create the directory holding git checkout
+  file:
+    name: /srv/{{ server_name }}/html/
+    state: directory
+
+- name: Clone the repo
+  git:
+    repo: https://github.com/gluster/gluster-bugs-webui.git
+    dest: /srv/{{server_name }}/html/
+    version: HEAD
```

Comment:
`version: HEAD` is the default. It is reasonable to assume this is what we want without being explicit